tox.ini

```diff
@@ -17,15 +19,8 @@ skip_missing_interpreters = true
 
 [testenv]
 description = Run pytest against the specified python version
-; if the tox environment name contains `dev` (e.g. `py39-dev`), pip install the
-; current package in editable mode (-e .)
-usedevelop =
-    dev: true
```

Comment:
this section is not needed, one can achieve the same thing by passing `tox --develop` option